### ISSUE TITLE
cif_core.dic: add the `_exptl_absorpt_special_details` data item

### ISSUE THREAD

**Issue by vaitkus** [COMCIFS/cif_core#234]
In the 267887 revision of the COD there are over 1000 files that contain the `_exptl_absorpt_special_details` data item, although, no such item is defined in the `CIF_CORE` dictionary.

My initial assumption was that this data item was simply used instead of the properly defined `_exptl_absorpt_process_details` data item. However, some additional analysis revealed that in ~800 cases out of the 1000, both the `_exptl_absorpt_special_details` and the `_exptl_absorpt_process_details` data items are defined and often with differing values. For example, COD entry 7060029 [1] contains the following data item values:

- `_exptl_absorpt_process_details`:

```
<i>SADABS</i> (Krause et al., 2015)
```
- `_exptl_absorpt_special_details`:
```
In addition to absorption, multi-scan techniques also correct other slowly 
varying changes in scale factor over reciprocal space, such as those caused 
by x-ray beam inhomogeneity, goniometer imperfection, crystal decomposition, 
crystal longer than the x-ray beam cross section, absorption by the crystal 
mount etc.  These account, by and large, for much of the difference between 
estimated values of Tmin and Tmax and the _exptl_absorpt_correction_ values. 
```

I am unsure where the `_exptl_absorpt_special_details` tag originated from, however, I have noticed that all of the affected files seem to be generated by SHELXL. What is more, I found some SHELX-related material online [2] that provides a usage example of the `_exptl_absorpt_special_details` data item (see slide "Comments on sample absorption correction").

[1] http://www.crystallography.net/cod/7060029.cif@258905
[2] https://chemistry.harvard.edu/files/chemistry/files/tutorial_using_shelx_for_neutrons.pdf

**Comment by jamesrhester** [comment]
I will pass this to the core DMG as clearly this is a data name that is useful, as shown by your ref [2].

**Comment by jamesrhester** [comment]
The core DMG support the addition of this data name to the dictionary. Given its origin, we will need to include the all underscore version as an alias.

**Comment by vaitkus** [comment]
Issue resolved by merging PR #234. Thanks, @rowlesmr.
